GiEnJul/Controllers/AdminController.cs

```diff
@@ -84,6 +85,26 @@ public async Task<FileStreamResult> DownloadExcelDeliveryLocationAsync(string lo
             using var wb = ExcelGenerator.Generate(_mapper.Map<IEnumerable<DeliveryExcel>>(connections));
             return wb.Deliver("leveranse_liste.xlsx");
         }
+
+        [HttpGet("connections/{location}")]
+        [Authorize(Policy = "ReadConnection")]
+        public async Task<IList<GetConnectionDto>> GetConnectionsByLocationAsync(string location)
+        {
+            var eventName = await _eventRepository.GetActiveEventForLocationAsync(location);
+            var completed = await _connectionRepository.GetAllConnectionsByLocation(eventName, location);
+            var Dto = _mapper.Map<List<GetConnectionDto>>(completed);
+            var suggestedGiver = await _giverRepository.GetSuggestedAsync(eventName, location);
+            var suggestedRecipient = await _recipientRepository.GetSuggestedAsync(eventName, location);
+            var suggested = new List<GetConnectionDto>();
+            foreach (var giver in suggestedGiver)
+            {
+                var mathedGiver = suggestedRecipient.FirstOrDefault(x => x.MatchedGiver == giver.RowKey);
+                suggestedRecipient.Remove(mathedGiver);
+                suggested.Add(_mapper.Map<GetConnectionDto>((giver, mathedGiver)));
+            }
+            Dto.AddRange(suggested);
```

Comment:
Is there a reason for not directly adding to this list and instead keeping 2 lists

Reply:
just did not think about it

Reply:
should be fixed with b6d17bd